code-improvements/greet.js

```diff
@@ -0,0 +1 @@
+const greet = (name) => `Hello${ name ?", " + name : ""}!`;
```

Comment:
The return value very hard to follow, check if you can refactor it.
You can ask your teammates too.